client/src/components/mobile/NavigationMobile.js

```diff
@@ -3,14 +3,15 @@
 import React, { useEffect, useState } from 'react';
 import '../../css_mobile/Navigation.css';
 import { NavLink } from 'react-router-dom';
-import { Drawer, Button, Icon } from 'antd';
+import { Drawer, Button, Icon, Menu } from 'antd';
 
 type Props = {
   authed: Boolean,
+  authRoleIsEquivalentTo: String => Boolean,
 };
 
 const NavMobile = (props: Props) => {
-  const { authed } = props;
+  const { authed, authRoleIsEquivalentTo } = props;
```

Comment:
I can't tell for sure, but is this unused?  If so we should remove it :P

Reply:
It will used to determine whether to show the NavLinks to admin and user page.